curricula/models.py

```diff
@@ -166,7 +166,11 @@ def jackfrost_urls(self):
         return urls
 
     def jackfrost_can_build(self):
-        return settings.ENABLE_PUBLISH and self.status == 2 and not self.login_required
+        can_build = settings.ENABLE_PUBLISH and self.status == 2 and not self.login_required and not self.curriculum.login_required
+        if not can_build:
+            logger.error('jackfrost_can_build returns %s. settings.ENABLE_PUBLISH: %s self.status: %s self.login_required: %s self.curriculum.login_required: %s' \
```

Comment:
Nitpick: shouldn't this be `warn` (or below) and not `error`?

Reply:
Happy to do warn. The reasoning was that as it stood, it seemed like methods that called jackfrost_can_build when jackfrost can't cause errors/weird behavior down the line, since they return None and I didn't see anything handling the None case anyway. But, technically this is not an error, so happy to change logging to warn